profiles/Ssarek-setup.yaml

```diff
@@ -92,6 +92,15 @@ vig: &vig
     cambrinth:
     - 20
     - 20
+vigpet: &vigpet
```

Comment:
💡 Since we're going to be pointing at this as an example of pet buffs being used, what do you think about renaming this to `vigor_pet_buff` (or something similarly more-verbose-but-clearer)?